src/EditorFeatures/Core/Implementation/CodeActions/CodeActionEditHandlerService.cs

```diff
@@ -119,14 +119,41 @@ public void Apply(Workspace workspace, Document fromDocument, IEnumerable<CodeAc
                 if (updatedSolution == oldSolution)
                 {
                     updatedSolution = applyChanges.ChangedSolution;
+                    var projectChanges = updatedSolution.GetChanges(oldSolution).GetProjectChanges();
+                    var changedDocuments = projectChanges.SelectMany(pd => pd.GetChangedDocuments());
+                    var changedAddiionalDocuments = projectChanges.SelectMany(pd => pd.GetChangedAdditionalDocuments());
+                    var changedFiles = changedDocuments.Concat(changedAddiionalDocuments);
 
-                    // check whether it contains only 1 or 0 changed documents
-                    if (!updatedSolution.GetChanges(oldSolution).GetProjectChanges().SelectMany(pd => pd.GetChangedDocuments()).Skip(1).Any())
+                    // 0 file changes
+                    if (!changedFiles.Any())
                     {
                         operation.Apply(workspace, cancellationToken);
                         continue;
                     }
 
+                    // 1 file change
+                    SourceText text = null;
```

Comment:
Nit: Consider moving ```text``` and the ```if (text != null)``` below into the ```if (!changedFiles.Skip(1).Any())``` body, since ```text``` isn't needed later and it might make it easier to see that this ```if`` has no side effects.